### ISSUE TITLE
train_script is missing

### ISSUE THREAD

**Issue by praveena2j** [kuhnkeF/ABAW2020TNT#9]

**Comment by kuhnkeF** [comment]
Hi praveena2j,
we do not plan to release the training code. However, the training is straightforward. We used multiple forward passes to get a bigger batch size (because gpu memory was too small).
opt.zero_grad()
for sub_batch in virtual_batch:
{
loss = model(sub_batch) / len(virtual_batch)
loss.backward()
}
opt.step()
Hope this helps. Everything is mentioned the paper.
